deployment/v3/mosip/prereg/install.sh

```
@@ -7,12 +7,20 @@ if [ $# -ge 1 ] ; then
 fi
 
 NS=prereg
+<<<<<<< HEAD
```

Shellcheck (0.11.0)

[error] Couldn't parse this here string. Fix to allow more checks.

(SC1073)